documentation/book/ref-sample-kafka-resource-config.adoc

```diff
@@ -0,0 +1,189 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-configuration.adoc
+
+[id='ref-kafka-bridge-producer-configuration-{context}']
+= Sample Kafka YAML configuration
+
+For help in understanding the configuration options available for your Kafka deployment, refer to sample YAML file provided here.
```

Comment:
I feel like we should clarify that these are:
* Just some of the configuration options
* these options are listed here because we think they are particularly important

We should probably also add a list of the important things and mention that they are also important for Kafka Connect, Mirror Maker etc.